From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@yamatoya__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <p>The agreement was validated by Microsoft and real humans are currently evaluating your PR.</p>

TTYL, AZPRBOT;

rmca14 commented:
@MightyPen 
Please review this PR and comment if I should merge it or not. Thanks.

MightyPen commented:
Yes, please DO Approve this P.R.# 4189.  Thanks.

mumian committed:
edit

mumian committed:
incorporate pm feedback

nitinme committed:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/master' into nitinme-hdi-updates

nitinme committed:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/master' into nitinme-hdi-updates

PatAltimore committed:
Remove classic portal reference

nitinme committed:
Adding screenshot for security package

nitinme committed:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/master' into nitinme-hdi-updates

nitinme committed:
adding image declaration

nitinme committed:
acrolinx improvements

mmacy committed:
[acr] integrate review comments

mmacy committed:
[acr] link fix

ShawnJackson committed:
edit pass: automation-invoke-runbook-from-omsla-alert

mgblythe committed:
Freshness for app/flow functions topics

rjagiewich committed:
edit pass: data-factory-compare-versions-2

rjagiewich committed:
Update compare-versions.md

ShawnJackson committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into automation-invoke-runbook-from-omsla-alert

ShawnJackson committed:
edit pass: automation-invoke-runbook-from-omsla-alert

Commit:
Portal and other updates.

Commit:
Fixes.

Commit:
Fix

Commit:
Updates.

PatAltimore committed:
Add delete tenant feedback; metadata update

cwatson-cat committed:
Added redirects for Backup topics to be retired

mmacy committed:
[acr] clarify 'ok to delete old storage acct'

PatAltimore committed:
Acrolinx; metadata

mgblythe committed:
Freshness for PowerApps + export topic

mmacy committed:
[acr] match fixed portal UI (singular 'registry')

Commit:
latencies08

mumian committed:
update

Commit:
latencies08

Commit:
latencies08

Commit:
latencies08

mmacy committed:
[acr] fix bookmark build warning

Commit:
latencies08

cwatson-cat committed:
Deleting files to be retired and redirected.

cwatson-cat committed:
Removed deleted files from Backup TOC.

ShawnJackson committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into automation-invoke-runbook-from-omsla-alert

mmacy committed:
[acr] minor formatting + 'managed' clarification

bsiva committed:
VMware to Azure PowerShell how to

MicrosoftGuyJFlo committed:
Updates to MFA Reports removal of PFWeb reports

ShawnJackson committed:
edit pass: automation-invoke-runbook-from-omsla-alert

cwatson-cat committed:
deleted media files for topics being retired.

bsiva committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr

bsiva committed:
TOC updates to add V2A PowerShell

bsiva committed:
Update support matrix to reflect support status for V2A PowerShell

Commit:
qfe release notes

Commit:
qfe release notes

bsiva committed:
TOC updates to link V2A PowerShell article to the automate section

MicrosoftGuyJFlo committed:
Updates to MFA Reports removal of PFWeb reports

MicrosoftGuyJFlo committed:
Updates to MFA Reports removal of PFWeb reports

mimig1 committed:
Cosmos DB: Removed partition key from portal for fixed size accts

mimig1 committed:
removed unneccessary files

shawntabrizi committed:
Updating Resource Health Documentation

shawntabrizi committed:
Update .gitignore

JeffGoldner committed:
Document the update procedures.

JeffGoldner committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr

bsiva committed:
Accrolinx comments addressed

JeffGoldner committed:
add missing script argument

bsiva committed:
fix formatting

MicrosoftGuyJFlo committed:
IP list for troubleshooting

PatAltimore committed:
Change metadata

ktoliver committed:
Merge pull request #30167 from shawntabrizi/shtabriz

Updating Resource Health Documentation for Mooncake

ktoliver committed:
Merge pull request #30159 from MicrosoftGuyJFlo/MFAReports

Updates to MFA Reports removal of PFWeb reports

Commit:
Merge pull request #30174 from MicrosoftGuyJFlo/SSPRNetwork

IP list for troubleshooting

JeffGoldner committed:
Add supported OS document

JeffGoldner committed:
Add supported OS document

Ja-Dunn committed:
Merge pull request #29547 from mmacy/acr-migrate-sku-update

[acr] (new) upgrade classic registry

ktoliver committed:
Merge pull request #30162 from bsiva/master

ASR V2A PowerShell documentation

iainfoulds committed:
Freshness updates and revisions

ktoliver committed:
Merge pull request #30169 from JeffGoldner/master

Add RP Update procedure sections

Ja-Dunn committed:
Merge pull request #30079 from PatAltimore/patricka-b2c-remove-old-portal

Remove classic portal reference from B2C

rjagiewich committed:
Merge pull request #30138 from cwatson-cat/12-15-Classic-RetireBackuptopics

Added redirects in master redirect file for Backup topics to be retired

iainfoulds committed:
Acrolinx updates

tfitzmac committed:
updated managed apps

garyericson committed:
Fix service metadata

mgblythe committed:
Freshness for OpenAPI doc

garyericson committed:
Fix manager metadata

PRMerger13 committed:
Merge pull request #30177 from iainfoulds/vmfreshness121517-batch3

VM freshness updates

garyericson committed:
Fix ms.reviewer metadata

garyericson committed:
Fix ms.reviewer metadata #2

ShawnJackson committed:
Update compare-versions.md

garyericson committed:
Fix ms.reviewer metadata #3

garyericson committed:
Fix ms.workload metadata

tfitzmac committed:
fixed code block

garyericson committed:
Fix ms.topic metadata

Commit:
Merge pull request #30181 from garyericson/12-15-metadata-manager

Fix manager metadata

PRMerger12 committed:
Merge pull request #30182 from garyericson/12-15-metadata-ms-reviewer

Fix ms.reviewer metadata

PRMerger18 committed:
Merge pull request #30184 from garyericson/12-15-metadata-ms-reviewer-2

Fix ms.reviewer metadata #2

PRMerger14 committed:
Merge pull request #30179 from garyericson/12-15-metadata-service

Fix service metadata

garyericson committed:
Merge branch 'master' into 12-15-metadata-ms-reviewer-3

Ja-Dunn committed:
Merge pull request #30109 from mgblythe/mb-azure-work

Freshness for app/flow functions topics

Ja-Dunn committed:
Merge pull request #30178 from tfitzmac/1215appid

updated managed apps

PRMerger13 committed:
Merge pull request #30186 from garyericson/12-15-metadata-ms-workload

Fix ms.workload metadata

PRMerger16 committed:
Merge pull request #30185 from garyericson/12-15-metadata-ms-reviewer-3

Fix ms.reviewer metadata #3

ShawnJackson committed:
Merge pull request #30112 from rjagiewich/data-factory-compare-versions-2

edit pass: data-factory-compare-versions-2

Commit:
Merge pull request #30188 from garyericson/12-15-metadata-ms-topic

Fix ms.topic metadata

garyericson committed:
Fix ms.reviewer metadata #4

garyericson committed:
Fix ms.reviewer metadata #5

v-thepet committed:
broken links

PRMerger12 committed:
Merge pull request #30190 from garyericson/12-15-metadata-ms-reviewer-4

Fix ms.reviewer metadata #4

PRMerger10 committed:
Merge pull request #30191 from garyericson/12-15-metadata-ms-reviewer-5

Fix ms.reviewer metadata #5

garyericson committed:
Fix manager metadata #2

Commit:
updates to doc

PRMerger17 committed:
Merge pull request #30193 from garyericson/12-15-metadata-manager-2

Fix manager metadata #2

Commit:
updates

Commit:
updates

PRMerger12 committed:
Merge pull request #30151 from MarkusVi/latencies08

latencies08

Commit:
qfe release notes

kevinlam1 committed:
Update logic-apps-loops-and-scopes.md

Removed old incorrect limits and added a reference to the limits and configuration page.

kevinlam1 committed:
Update logic-apps-loops-and-scopes.md

Fixed acrolinx issues.

kevinlam1 committed:
Update logic-apps-loops-and-scopes.md

Fixing more acrolynx issues.

kevinlam1 committed:
Update logic-apps-loops-and-scopes.md

More acrolynx issues.

PRMerger20 committed:
Merge pull request #30195 from kevinlam1/patch-15

Update logic-apps-loops-and-scopes.md

Commit:
updates to article

v-geberr committed:
new ui

v-geberr committed:
qs-create-new-app new ux

v-geberr committed:
about luis image

v-geberr committed:
new tutorial image for new UX

Commit:
Merge pull request #30192 from v-thepet/newfix

broken links

v-thepet committed:
15 broken links

Juliako committed:
Update api-management-custom-domain.md

Juliako committed:
added lnks to java and php

Commit:
Merge pull request #30199 from v-thepet/newfix

production bug: 15 broken links

cwatson-cat committed:
Removed reference to Classic portal in VM topic.

cwatson-cat committed:
Updated topic to raise acrolynx score.

PRMerger12 committed:
Merge pull request #30201 from Juliako/AMSUpdates

added lnks to java and php

cwatson-cat committed:
Second round of acrolynx fixes

PRMerger12 committed:
Merge pull request #30202 from cwatson-cat/12-16-ClassicClnVM

Removed reference to Classic portal in VM topic.

Commit:
no images in article \articles\virtual-machine-scale-sets\virtual-machine-scale-sets-autoscale-overview.md

Commit:
images not used in article \articles\virtual-machines\windows\aws-to-azure.md

Commit:
images not used in article \articles\storage\common\storage-powershell-guide-full.md

Commit:
images not used in article \articles\storage\blobs\storage-blob-scalable-app-upload-files.md

Commit:
images not used in article \articles\sql-database\sql-database-service-tiers.md

Commit:
images not used in article \articles\site-recovery\tutorial-vmware-to-azure-failover-failback.md

Commit:
images not used in article \articles\site-recovery\site-recovery-deployment-planner.md

Commit:
images not used in article \articles\service-fabric\service-fabric-understand-and-troubleshoot-with-system-health-reports.md

Commit:
images not used in article \articles\security-center\security-center-policies.md

Commit:
images not used in article \articles\security\azure-security-threat-modeling-tool.md

Commit:
images not used in article \articles\search\search-what-is-azure-search.md

Commit:
images not used in article \articles\search\search-filters.md

Commit:
images not used in \articles\operations-management-suite\operations-management-suite-solutions.md

Commit:
images not used in \articles\multi-factor-authentication\multi-factor-authentication-sdk.md

Commit:
images not used in \articles\mobile-engagement\mobile-engagement-dotnet-rest-service-api.md

Commit:
images not used in \articles\media-services\media-services-azure-media-clipper-getting-started.md

Commit:
images not used in \articles\machine-learning\team-data-science-process\move-sql-azure-adf.md

Commit:
images not used in \articles\logic-apps\logic-apps-enterprise-integration-xml.md

Commit:
images not used in \articles\logic-apps\logic-apps-enterprise-integration-agreements.md

Commit:
images not used in \articles\key-vault\key-vault-developers-guide.md

Commit:
images not used in \articles\iot-suite\iot-suite-what-are-preconfigured-solutions.md

Commit:
images not used in \articles\hdinsight\hdinsight-extend-hadoop-virtual-network.md

Commit:
images not used in \articles\data-lake-analytics\data-lake-analytics-u-sql-programmability-guide.md

Commit:
images not used in \articles\cosmos-db\tutorial-query-table.md

Commit:
images not used in \articles\cosmos-db\table-introduction.md

Commit:
images not used in \articles\vs-azure-tools-cloud-service-publish-set-up-required-services-in-visual-studio.md

Commit:
images not used in \articles\active-directory\active-directory-saas-access-panel-user-help.md

Commit:
images not used in \articles\active-directory\develop\active-directory-graph-api.md

Commit:
images not used in \articles\api-management\policy-samples.md

Commit:
images not used in \articles\automation\automation-verify-runas-authentication.md

Commit:
images not used in \articles\azure-government\documentation-government-plan-compliance.md

neilpeterson committed:
updated file extension

neilpeterson committed:
update file extension

Commit:
Merge pull request #30236 from neilpeterson/yaml-extension

updated file extension

GitHubber17 committed:
Merge pull request #30200 from Juliako/patch-1

Update api-management-custom-domain.md

GitHubber17 committed:
Merge pull request #30203 from SychevIgor/virtual-machine-scale-sets-autoscale-overview

images not used in article

GitHubber17 committed:
Merge pull request #30204 from SychevIgor/aws-to-azure

images not used in article \articles\virtual-machines\windows\aws-to-azure.md

GitHubber17 committed:
Merge pull request #30205 from SychevIgor/storage-powershell-guide-full

images not used in article \articles\storage\common\storage-powershell-guide-full.md

GitHubber17 committed:
Merge pull request #30206 from SychevIgor/storage-blob-scalable-app-upload-files

images not used in article \articles\storage\blobs\storage-blob-scalable-app-upload-files.md

GitHubber17 committed:
Merge pull request #30207 from SychevIgor/sql-database-service-tiers

images not used in article \articles\sql-database\sql-database-service-tiers.md

GitHubber17 committed:
Merge pull request #30208 from SychevIgor/utorial-vmware-to-azure-failover-failbac

images not used in article \articles\site-recovery\tutorial-vmware-to-azure-failover-failback.md

GitHubber17 committed:
Merge pull request #30209 from SychevIgor/site-recovery-deployment-planner

images not used in article \articles\site-recovery\site-recovery-deployment-planner.md

GitHubber17 committed:
Merge pull request #30210 from SychevIgor/service-fabric-understand-and-troubleshoot-with-system-health-reports

images not used in article \articles\service-fabric\service-fabric-understand-and-troubleshoot-with-system-health-reports.md

GitHubber17 committed:
Merge pull request #30211 from SychevIgor/security-center-policies

images not used in article \articles\security-center\security-center-policies.md

GitHubber17 committed:
Merge pull request #30212 from SychevIgor/azure-security-threat-modeling-tool

images not used in article \articles\security\azure-security-threat-modeling-tool.md

GitHubber17 committed:
Merge pull request #30214 from SychevIgor/search-what-is-azure-search

images not used in article \articles\search\search-what-is-azure-search.md

GitHubber17 committed:
Merge pull request #30215 from SychevIgor/search-filters

images not used in article \articles\search\search-filters.md

GitHubber17 committed:
Merge pull request #30216 from SychevIgor/operations-management-suite-solutions

images not used in \articles\operations-management-suite\operations-management-suite-solutions.md

GitHubber17 committed:
Merge pull request #30217 from SychevIgor/multi-factor-authentication-sdk

images not used in \articles\multi-factor-authentication\multi-factor-authentication-sdk.md

GitHubber17 committed:
Merge pull request #30218 from SychevIgor/mobile-engagement-dotnet-rest-service-api

images not used in \articles\mobile-engagement\mobile-engagement-dotnet-rest-service-api.md

GitHubber17 committed:
Merge pull request #30219 from SychevIgor/media-services-azure-media-clipper-getting-started

images not used in \articles\media-services\media-services-azure-media-clipper-getting-started.md

GitHubber17 committed:
Merge pull request #30220 from SychevIgor/move-sql-azure-adf

images not used in \articles\machine-learning\team-data-science-process\move-sql-azure-adf.md

GitHubber17 committed:
Merge pull request #30221 from SychevIgor/logic-apps-enterprise-integration-xml

images not used in \articles\logic-apps\logic-apps-enterprise-integration-xml.md

GitHubber17 committed:
Merge pull request #30222 from SychevIgor/logic-apps-enterprise-integration-agreements

images not used in \articles\operations-management-suite\operations-management-suite-solutions.md

GitHubber17 committed:
Merge pull request #30223 from SychevIgor/key-vault-developers-guide

images not used in \articles\key-vault\key-vault-developers-guide.md

GitHubber17 committed:
Merge pull request #30224 from SychevIgor/iot-suite-what-are-preconfigured-solutions

images not used in \articles\iot-suite\iot-suite-what-are-preconfigured-solutions.md

GitHubber17 committed:
Merge pull request #30225 from SychevIgor/hdinsight-extend-hadoop-virtual-network

images not used in \articles\hdinsight\hdinsight-extend-hadoop-virtual-network.md

GitHubber17 committed:
Merge pull request #30227 from SychevIgor/data-lake-analytics-u-sql-programmability-guide

images not used in \articles\data-lake-analytics\data-lake-analytics-u-sql-programmability-guide.md

GitHubber17 committed:
Merge pull request #30228 from SychevIgor/tutorial-query-table

images not used in \articles\cosmos-db\tutorial-query-table.md

GitHubber17 committed:
Merge pull request #30229 from SychevIgor/table-introduction

images not used in \articles\cosmos-db\table-introduction.md

GitHubber17 committed:
Merge pull request #30230 from SychevIgor/vs-azure-tools-cloud-service-publish-set-up-required-services-in-visual-studio

images not used in \articles\vs-azure-tools-cloud-service-publish-set-up-required-services-in-visual-studio.md

GitHubber17 committed:
Merge pull request #30231 from SychevIgor/active-directory-saas-access-panel-user-help

images not used in \articles\active-directory\active-directory-saas-access-panel-user-help.md

GitHubber17 committed:
Merge pull request #30232 from SychevIgor/active-directory-graph-api

images not used in \articles\active-directory\develop\active-directory-graph-api.md

GitHubber17 committed:
Merge pull request #30233 from SychevIgor/policy-samples

images not used in \articles\api-management\policy-samples.md

GitHubber17 committed:
Merge pull request #30234 from SychevIgor/automation-verify-runas-authentication

images not used in \articles\automation\automation-verify-runas-authentication.md

GitHubber17 committed:
Merge pull request #30235 from SychevIgor/documentation-government-plan-compliance

images not used in \articles\azure-government\documentation-government-plan-compliance.md

GitHubber17 committed:
Update release-notes-sprint-2.1.md

GitHubber17 committed:
Merge pull request #30194 from georgewallace/actiongroup

action groups : updates to doc

GitHubber17 committed:
Merge pull request #30160 from hning86/master

QFE release notes.

linda33wj committed:
Fix SQL DW JSON sample

PRMerger13 committed:
Merge pull request #30238 from linda33wj/master

Fix SQL DW JSON sample

Commit:
finish 1st guidance

Commit:
fix issues

Commit:
fix minor issues

Commit:
Merge branch 'master' into master

Commit:
delete dummy files

Commit:
add next steps

nabhishek committed:
updated 'remote access to intranet' section

Commit:
fix minor issues

PRMerger17 committed:
Merge pull request #30244 from nabhishek/patch-13

updated 'remote access to intranet' section

snehaamicrosoft committed:
change in scale limits

snehaamicrosoft committed:
table formatting

Commit:
add debugging tutorial

Commit:
fix minor issues

PRMerger14 committed:
Merge pull request #30245 from snehaamicrosoft/migrate

change in scale limits

Commit:
remove function guides

Commit:
update toc

Commit:
images not used in \articles\cognitive-services\Custom-Vision-Service\home.md

Commit:
minor

Commit:
minor

Commit:
added an FAQ

cephalin committed:
add link to script

PRMerger19 committed:
Merge pull request #30257 from cephalin/vnet

add link to script

ponatara committed:
Update site-recovery-support-matrix-to-azure.md

ponatara committed:
Update site-recovery-support-matrix-azure-to-azure.md

Commit:
Merge pull request #30260 from ponatara/patch-6

Update site-recovery-support-matrix-to-azure.md

Commit:
Merge pull request #30261 from ponatara/patch-7

Update site-recovery-support-matrix-azure-to-azure.md

PRMerger18 committed:
Merge pull request #30152 from mumian/1215-refresh

update

mumian committed:
update

PRMerger16 committed:
Merge pull request #30256 from nstulasi/saneppal/minorfix

added an FAQ

rastala committed:
Update known-issues-and-troubleshooting-guide.md

Describe notebook size limit for Workbench

Commit:
Incorporating Tamra feedback

PRMerger18 committed:
Merge pull request #30268 from rastala/master

Update known-issues-and-troubleshooting-guide.md

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into reviewStorageQS

Commit:
Update how-to-vscode-develop-csharp-module.md

Commit:
Update how-to-vscode-debug-csharp-module.md

Commit:
Update how-to-vscode-debug-csharp-module.md

rmca14 committed:
Merge pull request #30083 from nitinme/nitinme-hdi-updates

Updates for security package

Commit:
Update how-to-vscode-develop-csharp-module.md

Commit:
Acrolinx feedback

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into reviewStorageQS

itechedit committed:
Merge pull request #29285 from shizn/master

Add VSCode tutorials for IoT Edge

itechedit committed:
Merge pull request #29916 from mumian/1213-components

Enterprise Security Package

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

rmca14 committed:
Merge pull request #29944 from mumian/1213-djplan

update

itechedit committed:
Merge pull request #30125 from jimdial/patch-45

Portal and other updates.

v-alje committed:
Merge pull request #30272 from MicrosoftDocs/FromPublicRepo

From public repo

Ja-Dunn committed:
Merge pull request #30158 from ShawnJackson/automation-invoke-runbook-from-omsla-alert

edit pass: automation-invoke-runbook-from-omsla-alert

Commit:
Merge pull request #27284 from roygara/reviewStorageQS

Updating storage QS

itechedit committed:
Merge pull request #30166 from mimig1/12152017

Cosmos DB: Removed partition key from portal for fixed size accts

Lisaco88 committed:
Merge pull request #30198 from v-geberr/qs-create-new-app

new ui - qs-create-new-app

rjagiewich committed:
Merge pull request #30248 from SychevIgor/Custom-Vision-Service

images not used in \articles\cognitive-services\Custom-Vision-Service\home.md
